src/AVFoundation/AVTypes.cs

```diff
@@ -322,4 +322,13 @@ public struct AVSampleCursorChunkInfo {
 		[MarshalAs (UnmanagedType.I1)]
 		public bool HasUniformFormatDescriptions;
 	}
+
+	[NoTV, NoWatch, NoiOS, Mac (10,15)]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AVSampleCursorAudioDependencyInfo {
+		[MarshalAs (UnmanagedType.I1)]
+		public bool IsIndependentlyDecodable;
+
+		public nint PacketRefreshCount;
+	}
```

Comment:
That's incorrect. This file is not processed by the generator so the `[No*]` attributes are not affecting the output.

IOW `AVSampleCursorAudioDependencyInfo` is inside `Xamarin.iOS.dll` (and `*.TVOS.dll` and `*.WatchOS.dll`) even if not available on those platforms (check API diff).